myapp/views.py

```diff
@@ -29,8 +29,11 @@ def index(request):
         search_query = {}
 
         if "searchText" in kwargs:
-            search_query = {"searchText": kwargs['searchText']}
-            response = stock_api.list_stocks_names(kwargs['searchText'])
+            text = kwargs['searchText']
+            if "," in text:
+                text = text[:text.index(',')]
```

Comment:
Not sure how you thought of the comma ` "," ` edge case, but it is good that you considered it